### ISSUE TITLE
I want to get a culture trigger description, Am I not doing it right anywhere?

### ISSUE THREAD

**Issue by youxiaotian** [dahall/TaskScheduler#790]
I want to get a chinese description of the trigger。
but the ToString() seems not work.
![image](https://user-images.githubusercontent.com/35222088/49219309-d465e580-f40d-11e8-984a-b3fb54196547.png)
and in your document:
![image](https://user-images.githubusercontent.com/35222088/49219348-ef385a00-f40d-11e8-89ec-18a0aabcfbf6.png)

**Comment by youxiaotian** [comment]
I have get the newest code  and add the zh-CN  resources.

**Comment by youxiaotian** [comment]
Add to my question again.
I want to make a WindowsService which can scanning and collect information about all the task on the current computer.

**Comment by youxiaotian** [comment]
Thank you for your reply, your advice I have try early when I when I finished decompile the DLL, but it was dosn't work.
And then I have solved this problem about an hour ago.
it's caused by I didn't create a folder to contains the resouce.dll
like this:
before:
![image](https://user-images.githubusercontent.com/35222088/49235352-e73fe080-f434-11e8-87b5-29a29abc2cf5.png)
after:   fixed.
![image](https://user-images.githubusercontent.com/35222088/49235395-00489180-f435-11e8-85f7-200343cf090f.png)

I was going to close the question tomorrow, but I got your email, so I'd better get back to it now.
Thank you again for your reply.
